src/pages/index.html

```diff
@@ -1,22 +1,25 @@
 ---
 title: Home
 layout: layouts/page.html
-permalink: / 
+permalink: /
 draft: false
 eleventyExcludeFromCollections: false
-content_blocks: 
-    - _bookshop_name: generic/sample 
-      text: Hello World!
-
+content_blocks:
+    -
+        _bookshop_name: generic/icon
+        svg_icon: outline/arrow-long-up
+        big_icon: true
+        rounded_border: true
```

Comment:
Maybe we should discard these changes? It looks like a test output.

Reply:
I will make sure to delete it before merge.